src/server/cluster/cluster_family.cc

```diff
@@ -448,16 +437,17 @@ void DeleteSlots(const SlotSet& slots) {
 }
 
 void WriteFlushSlotsToJournal(const SlotSet& slots) {
-  if (slots.empty()) {
+  if (slots.Empty()) {
     return;
   }
 
   // Build args
   vector<string> args;
-  args.reserve(slots.size() + 1);
+  args.reserve(slots.Count() + 1);
   args.push_back("FLUSHSLOTS");
-  for (const SlotId slot : slots) {
-    args.push_back(absl::StrCat(slot));
+  for (SlotId slot = 0; slot <= SlotSet::kMaxSlot; ++slot) {
+    if (slots.Contains(slot))
+      args.push_back(absl::StrCat(slot));
```

Comment:
nit(also not this PR): Wow, this should use slot ranges instead 😵

Reply:
I also think so, but It will change the interface of the command, so I decided not doing it now